_maps/shuttles/shiptest/libertatia.dmm

```diff
@@ -0,0 +1,2414 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"au" = (
+/obj/effect/turf_decal/lumos/tile/neutral/fulltile,
+/obj/machinery/computer/autopilot,
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/effect/turf_decal/box,
+/turf/open/floor/pod/light,
+/area/ship/bridge)
+"az" = (
+/obj/effect/turf_decal/lumos/tile/neutral/fulltile,
+/obj/machinery/computer/helm,
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/effect/turf_decal/box,
+/turf/open/floor/pod/light,
+/area/ship/bridge)
+"aW" = (
+/obj/effect/turf_decal/lumos/tile/neutral/fulltile,
+/obj/machinery/holopad/emergency/command,
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/machinery/button/door{
+	id = "pirateshutters";
+	name = "Window Lockdown";
+	pixel_x = -24;
+	pixel_y = -5
+	},
+/obj/machinery/button/door{
+	id = "piratecargo";
+	name = "Blast Door Control";
+	pixel_x = -24;
+	pixel_y = 5
+	},
+/obj/machinery/light/small{
+	dir = 8;
+	pixel_y = -10
+	},
+/turf/open/floor/pod/light,
+/area/ship/bridge)
+"bb" = (
+/obj/effect/turf_decal/lumos/tile/neutral/fulltile,
+/obj/structure/chair/comfy/shuttle{
+	dir = 1
+	},
+/obj/structure/railing,
+/obj/item/radio/intercom/wideband{
+	pixel_x = 28;
+	pixel_y = 5
+	},
+/obj/item/radio/intercom{
+	pixel_x = 28;
+	pixel_y = -10
+	},
+/obj/effect/landmark/start/captain,
+/obj/effect/decal/cleanable/dirt/dust,
+/turf/open/floor/pod/light,
+/area/ship/bridge)
+"br" = (
+/obj/effect/turf_decal/bot,
+/obj/structure/closet/crate/critter{
+	name = "DANGER: DO NOT OPEN"
+	},
+/mob/living/simple_animal/bot/secbot{
+	desc = "The box said "do not open" for a reason.";
```

Comment:
```suggestion
	desc = "The box said \"do not open\" for a reason.";
```

Reply:
fffffff of course i forgot to escape those